app/src/main/java/ru/prometeydev/movie/FragmentMoviesDetails.kt

```diff
@@ -44,21 +44,21 @@ class FragmentMoviesDetails : Fragment() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         loadData()
 
-        movie?.let {
+        movie?.let { mov ->
```

Comment:
Why not to `movie`?

Reply:
Because the private var movie is already there, so as not to get confused.

Reply:
Sorry, for long reply, I would probably use something like `movieValue` or so.